File: tys-application/tys-commons/src/main/java/co/com/mintrabajo/tys/commons/domain/DatosSolicitudTramite.java
package co.com.mintrabajo.tys.commons.domain;

import java.util.List;

import co.com.mintrabajo.tys.commons.domain.messages.ObtenerDatosSeccionalResponse;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@AllArgsConstructor
@Builder(builderMethodName = "newInstance")
public class DatosSolicitudTramite {

	private CorrespondenciaTramite correspondencia;
	private SolicitudTramite solicitudTramite;
	private DatosEmpresa empresa;
	private DatosCondicionVinculacion condicionVinculacion;
	private List<DocumentoTramite> listDocumentos;
	private ObtenerDatosSeccionalResponse dependencias;
	private List<JustaCausa> listaJustaCausa;
	
	private List<OrganizacionSindical> listaOrganizacionesSindicales;

	private DatosFormacionAdolecente datosFormacion;
	private DatosEscenarioPractica datosEscenarioPractica;
	private DatosVinculacionPracticasAdolecente datosVinculacion;
	private String strJsonDatosBasicosPdf;
	public DatosSolicitudTramite(){
		
	}
}

File: tys-spa/ciudadano/src/app/domain/datos-solicitud-tramite.ts
import {CorrespondenciaTramite} from './correspondencia-tramite';
import {SolicitudTramite} from './solicitud-tramite';
import {DatosEmpresa} from './datos-empresa';
import {DatosCondicionVinculacion} from './datos-condicion-vinculacion';
import {DocumentoTramite} from './documento-tramite';
import {DatosFormacionAdolecente} from './datosFormacionAdolecente';
import {DatosEscenarioPractica} from './datosEscenarioPractica';
import {DatosVinculacionPracticasAdolecente} from './datosVinculacionPracticasAdolecente';

export interface DatosSolicitudTramite {
  correspondencia: CorrespondenciaTramite;
  solicitudTramite: SolicitudTramite;
  empresa: DatosEmpresa;
  condicionVinculacion?: DatosCondicionVinculacion;
  listDocumentos: Array<DocumentoTramite>;
  listaJustaCausa: Array<any>;
  datosFormacion: DatosFormacionAdolecente;
  datosEscenarioPractica: DatosEscenarioPractica;
  datosVinculacion: DatosVinculacionPracticasAdolecente;
  listaOrganizacionesSindicales: Array<any>;
  strJsonDatosBasicosPdf: string;
}
